pkg/ccip/chainaccessor/ton_accessor.go

```diff
@@ -648,7 +647,8 @@ func (a *TONAccessor) GetChainFeePriceUpdate(ctx context.Context, selectors []cc
 	for _, selector := range selectors {
 		result, err := a.client.RunGetMethod(ctx, block, addr, "destinationChainGasPrice", uint64(selector))
 		// The plugin is built with EVM behaviour in mind: if a value doesn't exist the zero value is returned
-		if execError, ok := err.(ton.ContractExecError); ok && execError.Code == common.ErrUnknownDestChainSelector { //nolint:errorlint // we're guaranteed to get unwrapped error here
+		if execError, ok := err.(ton.ContractExecError); ok && execError.Code == 24814 { //nolint:errorlint // we're guaranteed to get unwrapped error here
```

Comment:
added todo but still don't like this. The error hard-coded here in order to match on-chain definition. There's a [discrepancy](https://chainlink-core.slack.com/archives/C08KVCWAZUG/p1761933305938969?thread_ts=1761932964.876469&cid=C08KVCWAZUG) of the error code definition